src/Pages/Overview/DisplayProfile.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import './App.css'
+const enums = require('../../Enums.js')
+
+let toggle = false
+let toggleDelete = false
+let userToDelete = null
+let userToEdit = null
+
+// displaying children of member-board
+export function displayBoard (user, index) {
```

Comment:
Make `displayboard` it's own react component, and give it a better name for example `<OverviewProfile />`

Reply:
i will use hook